revocation/crl/fetcher.go

```diff
@@ -0,0 +1,158 @@
+// Copyright The Notary Project Authors.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package crl provides Fetcher and Cache interface and implementations for
+// fetching CRLs.
```

Comment:
The package does not contain Cache implementations, right?
```suggestion
// Package crl provides Fetcher and Cache interface for
// fetching CRLs.
```

Reply:
Updated.